sound/soc/sof/intel/hda-dsp.c

```diff
@@ -442,10 +443,7 @@ static int hda_dsp_set_D0_state(struct snd_sof_dev *sdev,
 	/* fallback to the previous register value */
 	value = value ? 0 : SOF_HDA_VS_D0I3C_I3;
 
-	/*
-	 * This can fail but return the IPC error to signal that
-	 * the state change failed.
-	 */
+	/* This can fail but we should return the IPC error */
```

Comment:
I preferred the older comment, more self-explanatory.

Reply:
oops not intentional. will fix.